src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js

```diff
@@ -133,17 +134,13 @@ export const DatasetMetadata = () => {
             labels={tagsEditorLabels}
             values={dataset?.tags}
             readOnly={false}
-            onChange={(newTags) => {
-              // TODO: add saga & call back-end to update dataset tags
-            }}
+            onChange={(newTags) => updateDataset(datasetId, { tags: newTags })}
           />
         </Grid>
         <DescriptionEditor
           value={dataset?.description}
           readOnly={false}
-          onChange={(newDescription) => {
-            // TODO: add saga & call back-end to update dataset description
-          }}
+          onChange={(newDescription) => updateDataset(datasetId, { description: newDescription })}
```

Comment:
In this context, we can easily retrieve the index of the dataset (that is the index of the currently selected dataset): we must add this argument to `updateDataset` so it can be used in the saga, and then transferred to the reducer